morecantile/models.py

```diff
@@ -233,21 +233,60 @@ def _invert_axis(self) -> bool:
         """Check if CRS has inverted AXIS (lat,lon) instead of (lon,lat)."""
         return crs_axis_inverted(self.crs)
 
+    @classmethod
+    def from_v1(cls, **kwargs) -> "TileMatrixSet":
```

Comment:
what the advantage of using `**kwargs: Any` over `tms: Dict` ?

Reply:
None at all. Changed in current commit